doc/2/api/controllers/server/metrics/index.md

```diff
@@ -0,0 +1,75 @@
+---
+code: true
+type: page
+title: metrics
+---
+
+# metrics
+
+
+
+Returns inner metrics directly from the current Kuzzle node core components.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/_metrics
+Method: GET
+```
+
+### Other protocols
+
+```js
+{
+  "controller": "server",
+  "action": "metrics"
+}
+```
+
+---
+
+## Response
+
+Returns the found statistic snapshots in the following format:
+
+- `funnel`: current node API funnel metrics
```

Comment:
`funnel` is an internal name. We could think about sometime more explicit like `api`